_includes/tutorials/anomaly-detection/ksql/markup/dev/create-transactions-stream.adoc

```diff
@@ -0,0 +1,5 @@
+Likewise, you'll need a ksqlDB stream and Kafka topic to represent transaction events. The transaction information includes the identifier, the user sending the money, the time of the transaction, and the name of the recipient. Since this data represents a historical sequence of events, a stream is most appropriate.
```

Comment:
Note: the _order_ of fields in the description does not match the order of fields in the ksqlDB statement. For a user following along, can we update the fields or the query so they are in sync?

Reply:
100% agree, this has been changed to match the order of the fields in the ksqlDB statement.